File: eMAR from Winston Laptop/Presentation/C_Development/Work/Ray/emar/src/app/interfaces/allergy.ts
export interface Allergy {
  actionStatus: string;
  allergyDrugId: string;
  alternateName: string;
  comment: string;
  drugId: string;
  id: number;
  informationSource: string;  // such as PulseCheck or Interface
  informationSourceCode: string;
  internalDrugId?: string;
  isActive: boolean;
  name: string;
  ndc: string;
  parentDrugId: string;
  parentDrugName: string;
  patientId: number;
  reaction: string;
  schedule: string;
  severity: string;
  source?: string; // such as Patient, Parent, etcetera
}

File: eMAR from Winston Laptop/WebAPI/API/C_Development/Work/Winston/PulseCheck.Domain/Allergy.cs
using System;
using System.Collections.Generic;

namespace PulseCheck.Domain
{
    public class Allergy
    {
        public string Name { get; set; }
        public Identifier Reaction { get; set; }
        public Identifier Severity { get; set; }
        public Identifier Source { get; set; }
        public string Comment { get; set; }
        public Status Status { get; set; }
        public Status ActionStatus { get; set; }
        public MinimalUser User { get; set; }
        public MinimalUser UserChg { get; set; }
        public DateTime? DateAdd { get; set; }
        public DateTime? DateChg { get; set; }
        public List<Identifier> Codes { get; set; }
    }
}